util/crlite_enrollment_id.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
```

Comment:
Like I said in the other review, this should live where it's actually used. There's precedent for having utility scripts alongside tests in mozilla-central.

Reply:
OK, I'll put a copy there.